styles/index.scss

```diff
@@ -0,0 +1,12 @@
+@import 'settings';
+@import 'mixins';
+@import 'grid';
+@import 'base';
+@import 'layout';
+
+@import 'components/modal/styles.scss';
+@import 'components/layout/header/styles.scss';
+@import 'components/layout/content/styles.scss';
+@import 'components/layout/footer/styles.scss';
+@import 'components/progress-bar/styles.scss';
+@import 'components/icon/styles.scss';
```

Comment:
I prefer to import the scss directly on the js component so we keep everything at a component level